packages/location-field/src/index.tsx

```diff
@@ -0,0 +1,758 @@
+            id="otpUi.LocationField.stops"
+          />
+        </S.MenuItem>
+      ),
+      stopFeatures.map(feature => renderFeature(itemIndex++, feature)),
+
+      transitFeaturesPresent && otherFeatures.length > 0 && (
+        <S.MenuItem bgColor="#333" header centeredText key="other-header">
+          <FormattedMessage
+            description="Text for header above the 'other'"
+            id="otpUi.LocationField.other"
+          />
+        </S.MenuItem>
+      ),
+      otherFeatures.map(feature => renderFeature(itemIndex++, feature))
+    );
+  }
+
+  /* 2) Process nearby transit stop options */
+  if (nearbyStops.length > 0 && !suppressNearby) {
+    // Add the menu sub-heading (not a selectable item)
+    menuItems.push(
+      <S.MenuItem header centeredText key="ns-header">
+        <FormattedMessage
+          description="Text for header above nearby stops"
+          id="otpUi.LocationField.nearby"
+        />
+      </S.MenuItem>
+    );
+
+    // Iterate through the found nearby stops
+    menuItems = menuItems.concat(
+      nearbyStops.map(stopId => {
+        // Construct the location
+        const stop = stopsIndex[stopId];
+        const stopLocation = {
+          id: stopId,
+          lat: stop.lat,
+          lon: stop.lon,
+          name: stop.name
+        };
+
+        // Create the location selected handler
+        const locationSelected = () => {
+          setLocation(stopLocation, "STOP");
+        };
+
+        // Add to the selection handler lookup (for use in onKeyDown)
+        locationSelectedLookup[itemIndex] = locationSelected;
+
+        // Create and return the option menu item
+        const option = (
+          <TransitStopOption
+            isActive={itemIndex === activeIndex}
+            key={optionKey++}
+            onClick={locationSelected}
+            stop={stop}
+            stopOptionIcon={stopOptionIcon}
+          />
+        );
+        itemIndex++;
+        return option;
+      })
+    );
+  }
+
+  /* 3) Process recent search history options */
+  if (sessionSearches.length > 0) {
+    // Add the menu sub-heading (not a selectable item)
+    menuItems.push(
+      <S.MenuItem header centeredText key="ss-header">
+        <FormattedMessage
+          description="Text for header above recently searched items"
+          id="otpUi.LocationField.recentlySearched"
+        />
+      </S.MenuItem>
+    );
+
+    // Iterate through any saved locations
+    menuItems = menuItems.concat(
+      sessionSearches.map(sessionLocation => {
+        // Create the location-selected handler
+        const locationSelected = () => {
+          setLocation(sessionLocation, "SESSION");
+        };
+
+        // Add to the selection handler lookup (for use in onKeyDown)
+        locationSelectedLookup[itemIndex] = locationSelected;
+        // Create and return the option menu item
+        const option = (
+          <Option
+            icon={sessionOptionIcon}
```

Comment:
Sort props (multiple instances)

Reply:
Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc